### ISSUE TITLE
feature: add option to use custom corpus file

### ISSUE THREAD

**Issue by dune73** [coreruleset/go-ftw#388]
With the URL encoding issue on 920220/920221 PL1 (https://github.com/coreruleset/coreruleset/issues/3926) we have a file that has a corpus that brings an FP with every payload. Running ftw with this very corpus would be useful.

**Comment by fzipi** [comment]
Yes, we have thought about this already for using a `File` corpus

**Comment by dune73** [comment]
Custom Corpus example file in original corpus format with line number and tab.
[custom-corpus.txt](https://github.com/user-attachments/files/17617648/custom-corpus.txt)
